### ISSUE TITLE
Sapper compatability 

### ISSUE THREAD

**Issue by bkd** [okrad/svelte-progressbar#9]
in LinearProgressBar using **window** to determine legacy browser breaks things. Only a minor change for that needed for Sapper to work, just check if window is available first

In Stop.svelte, this breaks again with overallPerc not subscribed to anything in the store - you dont really need to use $ for the values unless you are setting them, so we can do the following
```
<stop
  offset="{Math.round((serie.store.prevOffset * 100) / overallPerc)}%"
  stop-color="{serie.color}"
  class="{serie.cls}"
></stop>
<stop
  offset="{Math.round((serie.store.offset * 100) / overallPerc)}%"
  stop-color="{serie.color}"
  class="{serie.cls}"
></stop>
```

**Comment by okrad** [comment]
As for the first issue, it's now fixed.

As for the second one, I'm not sure you can get rid of the "$", and in fact if I remove them I get an _Unexpected value NaN% parsing offset attribute._ error in the developer tools console.

What's the exact error you get?

**Comment by albertms10** [comment]
I still get the following error using Sapper when the page first loads:

```
ReferenceError: window is not defined
    at node_modules/@okrad/svelte-progressbar/src/LinearProgressBar.svelte:41:5
    ...
```

What I’m using is the following `ProgressBar` inside a Svelte component:

```svelte
<ProgressBar
  style="thin"
  height={2}
  series={[80]}
  thresholds={[{ till: 50, color: '#800000' }, { till: 100, color: '#008000' }]} />
```

**Comment by okrad** [comment]
Thank you @albertms10, now it should be definitely fixed with the 1.8.0 version... Can you confirm it?

**Comment by albertms10** [comment]
Yes, definitely this explicit check was needed! Thank you!
